public/app/plugins/datasource/elasticsearch/datasource.ts

```diff
@@ -296,6 +303,12 @@ export class ElasticDatasource
     }
 
     const queryInterpolated = this.interpolateLuceneQuery(queryString);
+
+    let finalQuery = queryInterpolated;
+    filters.forEach(filter => {
+      finalQuery = addAddHocFilter(finalQuery, filter);
+    })
```

Comment:
```suggestion
    const finalQuery = this.addAdHocFilters(queryInterpolated, filters);
```

I think this can be simplified to the code above - to use  `this.addAdHocFilters` that does the same thing.